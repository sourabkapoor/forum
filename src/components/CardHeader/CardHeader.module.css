.card-header__top{
  /* padding: 0 12px 12px; */
  border-radius: 3px 3px 0 0;
  line-height: 12px;
  letter-spacing: 0.5px;
  font-weight: 700;
  padding-bottom: 12px;
}

.card-header__title {
  color: #818384;
  font-weight: 700;
  line-height: 18px;
  text-transform: none;
}