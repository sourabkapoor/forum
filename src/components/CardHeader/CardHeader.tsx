import CSSModules from "react-css-modules";
import styles from "./CardHeader.module.css";

const CardHeader: React.FC = () => {
  return (
    <div styleName="card-header__top">
      <h2 styleName="card-header__title">About Community</h2>
    </div>
  );
};

export default CSSModules(CardHeader, styles, {
  allowMultiple: true,
  handleNotFoundStyleName: "log",
});
