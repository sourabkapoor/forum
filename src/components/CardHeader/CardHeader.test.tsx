import {
  queryByPlaceholderText,
  queryByTestId,
  render,
  screen,
} from "@testing-library/react";
import "@testing-library/jest-dom";
import userEvent from "@testing-library/user-event";
import each from "jest-each";
import CardHeader from "./CardHeader";

describe("CardHeader", () => {
  test("snapshot", () => {
    const { asFragment } = render(<CardHeader />);

    expect(asFragment()).toMatchSnapshot();
  });
});
