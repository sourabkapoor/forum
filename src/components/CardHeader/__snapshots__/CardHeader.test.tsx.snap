// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardHeader snapshot 1`] = `
<DocumentFragment>
  <div
    class="card-header__top"
  >
    <h2
      class="card-header__title"
    >
      About Community
    </h2>
  </div>
</DocumentFragment>
`;
